src/olympia/lib/es/tests/test_commands.py

```diff
@@ -26,6 +26,10 @@ def setUp(self):
 
         self.addons = []
         self.expected = self.addons[:]
+        # Monkeypatch Celerys ".get()" inside async task error
+        # until https://github.com/celery/celery/issues/4661 (which isn't just
+        # about retries but a general regression) fixed.
```

Comment:
As above... but this test is doing threads to mimic a reindexation taking place so maybe it should use "real" celery and not eager mode ?

Reply:
Indeed, but tbh, I'd like to tackle that in a separate step. I spend quite some time to move this to a version where it uses a separate worker but continuously ran into transaction barriers. Also keeps the patch small (hoping to use that as an excuse…)